api/v1/controllers/bots.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
```

Comment:
What about endpoints for adding/removing/listing who is authorized to update or delete a Bot?